src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -40,8 +40,8 @@ pub struct CrashReport {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub minimized_stack_function_names_sha256: Option<String>,
 
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub minimized_stack_function_lines: Vec<String>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub minimized_stack_function_lines: Option<Vec<String>>,
```

Comment:
Confirming: this can't cause any breakage for Python consumers, because we're simply adopting the (nullable) field definition already used in the corresponding [model](https://github.com/microsoft/onefuzz/blob/main/src/pytypes/onefuzztypes/models.py#L218-L223).

Reply:
Correct.